src/parametrization_tendencies.jl

```diff
@@ -0,0 +1,13 @@
+struct parametrization_tendencies{T<:AbstractFloat}
+
+#Just a placeholder
+
+end
+
+"""
+Compute physical parametrization tendencies
+"""
+function phypar{T}(utend, vtend, ttend, qtend) where {T<:AbstractFloat}
```

Comment:
Generator functions must have the same name as the generated struct, otherwise the `{T}` notation causes an error. They basically extend the default generating function of the struct (which just takes all elements in the structs as arguments) with an additional method (in the multiple-dispatch sense).

Reply:
Good catch. Fixed